Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
+    int i = 2;
+    do
+    {
+        if ((value % i == 0 || value < 2) && value!=2)
+        {
+            isPrimeInd = false;
+            break;
+		}
+        i++;
+    }while (i<value);
```

Comment:
we're starting check from 2, so we don't need to check anything higher than value/2 etc.
We also don't need to check value/3, because 3 will be checked first (and no multiple of 2 or 3 needs to be checked either) etc.
Following this train of thought, Actually the highest divisor we need to check is sqrt(value), 
because all higher ones would mean it is multiplied by a number already checked in one of the previous iterations
* this is the basic idea behind https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
** this entire comment is a nitpick and you don't actually need to implement the Sieve to make me happy ;)